### ISSUE TITLE
Publish binary release to Homebrew

### ISSUE THREAD

**Issue by pditommaso** [seqeralabs/tower-cli#128]
It would be nice to push binary releases to *Homebrew*. 

It should come for free using https://jreleaser.org/guide/latest/distributions/native-image.html

**Comment by jordeu** [comment]
Why not `sdkman`? it is more multiplatform.

**Comment by pditommaso** [comment]
I think Sdkman users intersection with our Nextflow/Tower is an empty set 😄

**Comment by ewels** [comment]
How about `conda`? 👀

**Comment by pditommaso** [comment]
Yeah, I was mentioning Homebrew because it comes out of the box with the release tool we are using. But Conda makes a lot of sense. 

Anyway it's just static binary, download it and run.
